docs/workforce-management-framework/topics/pro-enabling-logging.adoc

```diff
@@ -0,0 +1,27 @@
+= Enabling logging
+
+By default, all {WFM-RC-NameLong} framework modules will not log any information to the console.
+Framework users can enable logger by using one of the provided implementations or the
+custom implementation that will extend our interface.
+
+Supported implementations:
+
+|====
+| Implementation | Usage
+
+|BunyanLogger |Implements Bunyan and is used for server side logging
+|ClientLogger |Implements LogLevel and is used for client (web and mobile) logging
+|====
+
```

Comment:
Should we included sample of Console too?

[source,javascript]
----
// Import logger implementation
import {ConsoleLogger, logger, setLogger} from '@raincatcher/logger';

// Instaniate logger
const log = new ConsoleLogger();;
setLogger(log);
----
+
. To log a comment, add the following code _after_ the code directly above:
+
[source,javascript]
----
logger.info('This log will render with ConsoleLogger');
----